dashboard/app/controllers/levels_controller.rb

```diff
@@ -141,7 +141,9 @@ def show
   # GET /levels/1/edit
   def edit
     # Make sure that the encrypted property is a boolean
-    @level.properties['encrypted'] = @level.properties['encrypted'].to_bool if @level.properties['encrypted']
+    if @level.properties['encrypted']&.is_a?(String)
+      @level.properties['encrypted'] = @level.properties['encrypted'].to_bool
+    end
```

Comment:
to_bool is defined here: https://github.com/code-dot-org/code-dot-org/blob/65a46d3b8cb8eb616dc8fb5e5ab63eeebf80bcee/lib/cdo/pegasus/string.rb#L17-L16